docs/final report/chapters/engineering-reflection.tex

```diff
@@ -1,12 +1,32 @@
 \chapter{Engineering Reflection} %MAX 1 PAGE
 
-Introduce the chapter, explain what will be descibed. 2 PAGES
+In this section we will describe how the engineering process went. We discuss how we have used scrum to develop the product. Furthermore we will discuss the GitHub pull requests and how we ensured that our code had good quality. As last we will discuss the feedback we got from SIG.
+
+\section{Scrum}
+We used the scrum work flow to develop the product. On Friday we created a plan for the coming week. Every week-day we started with a short daily-meeting. In that meeting we discussed what we had done, what we were going to do and which problems we encountered. At Friday our sprint ended. We created a sprint-reflection and we gave a demo to the client. Next we started a new sprint. 
```

Comment:
~~On~~ Friday.

Streepje tusssen daily en meeting weg.

we **had** done samentrekken.

we **were** going samentrekken.

we encountered -> we'd encountered.

At Friday -> On Friday

Streepje tussen sprint en reflection weg.

Next -> After which. we started **on** ipv. **a**.